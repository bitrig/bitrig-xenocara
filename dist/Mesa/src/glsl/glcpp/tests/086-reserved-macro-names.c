#define __BAD reserved
#define GL_ALSO_BAD() also reserved
