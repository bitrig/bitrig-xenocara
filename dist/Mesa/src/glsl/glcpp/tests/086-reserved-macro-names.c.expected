0:1(10): preprocessor error: Macro names starting with "__" are reserved.

0:2(9): preprocessor error: Macro names starting with "GL_" are reserved.




